### ISSUE TITLE
First Experiment (PENN Treebank)

### ISSUE THREAD

**Issue by efittschen** [comp-int-hum/poetry-war-effects#18]
Generate a graph of model performance given a certain percentage of PENN tree bank data.
Generate this graph by training 3 different seeds of models over the PENN tree bank and checkpointing every 5 percent. 

Run this same graph given different amounts of pre-pretrainng data - the hope is that the loss cuve will fall earlier.
